queries.sql

```diff
@@ -54,4 +54,27 @@ UPDATE ANIMALS SET species='Pokemon' WHERE NAME='Squirtle';
 //Update the skipped Squirtle animal data weight_kg column;
 UPDATE ANIMALS SET weight_kg = weight_kg*-1 WHERE name='Squirtle';
 
+//Write queries (using JOIN) to answer the following questions
+//:What animals belong to Melody Pond?
+SELECT name FROM ANIMALS JOIN OWNERS ON ANIMALS.owner_id=OWNERS.id WHERE TRIM(OWNERS.full_name)='Melody Pond';
+//List of all animals that are pokemon (their type is Pokemon).
+SELECT ANIMALS.name FROM ANIMALS JOIN SPECIES ON ANIMALS.species_id=SPECIES.id WHERE ANIMALS.species_id=1;
```

Comment:
- [ ] This query returns all the Digimon instead of Pokemon, this is most likely due to a small mistake in the `data.sql` file where you didn't set the value for Pokemon species.
https://github.com/ismayito/vet-clincic-database/blob/624048eab57177ba5b82c8bfe5feb6c7b0fd1856/data.sql#L29-L32